public/assets/.sprockets-manifest-4f5db62d128c43c9bb0ba2d715df1fa5.json

```diff
@@ -0,0 +1 @@
+{"files":{"manifest-dad05bf766af0fe3d79dd746db3c1361c0583026cdf35d6a2921bccaea835331.js":{"logical_path":"manifest.js","mtime":"2021-06-03T15:36:13-04:00","size":3,"digest":"6a3cf5192354f71615ac51034b3e97c20eda99643fcaf5bbe6d41ad59bd12167","integrity":"sha256-ajz1GSNU9xYVrFEDSz6Xwg7amWQ/yvW75tQa1ZvRIWc="},"favicon-5165f4a6f467d4decc14352b9cd3a2be0e1010144f5bc0d611583f9dceb9e4d6.ico":{"logical_path":"favicon.ico","mtime":"2021-06-03T15:36:13-04:00","size":4710,"digest":"381ee179a3be9bfee4c78b543207575e2f8267fa910a2a29d9c1dfd936604341","integrity":"sha256-OB7heaO+m/7kx4tUMgdXXi+CZ/qRCiop2cHf2TZgQ0E="},"trans-header-large-f32bf6d302aeb1541f9e62d1e1309f4b6b20ec1eaf05e24a8a63c7b37bd134b1.png":{"logical_path":"trans-header-large.png","mtime":"2021-06-03T15:36:13-04:00","size":129004,"digest":"98a1f8fdc4da32cc142910bdfa1197f11cb53647a13ca1f44a31eb7420e56944","integrity":"sha256-mKH4/cTaMswUKRC9+hGX8Ry1NkehPKH0SjHrdCDlaUQ="},"wordmark-5a854e85808deea210825adb8846ce075251470593029f4e1c28f4a5c0fa42ef.svg":{"logical_path":"wordmark.svg","mtime":"2021-06-03T15:36:13-04:00","size":6555,"digest":"2815e9da48f6243e31bc64959fee770bd4348c49294c02eefe90a2e51034c785","integrity":"sha256-KBXp2kj2JD4xvGSVn+53C9Q0jEkpTALu/pCi5RA0x4U="}},"assets":{"manifest.js":"manifest-dad05bf766af0fe3d79dd746db3c1361c0583026cdf35d6a2921bccaea835331.js","favicon.ico":"favicon-5165f4a6f467d4decc14352b9cd3a2be0e1010144f5bc0d611583f9dceb9e4d6.ico","trans-header-large.png":"trans-header-large-f32bf6d302aeb1541f9e62d1e1309f4b6b20ec1eaf05e24a8a63c7b37bd134b1.png","wordmark.svg":"wordmark-5a854e85808deea210825adb8846ce075251470593029f4e1c28f4a5c0fa42ef.svg"}}
```

Comment:
I'm fairly certain that all of these `public/assets` files are asset hashes from the compilation options you switched on (see above) that shouldn't be here.